x/observer/keeper/msg_server_update_crosschain_flags_test.go

```diff
@@ -41,6 +41,10 @@ func TestMsgServer_UpdateCrosschainFlags(t *testing.T) {
 				RetryInterval:           time.Minute * 42,
 				GasPriceIncreasePercent: 42,
 			},
+			BlockHeaderVerificationFlags: &types.BlockHeaderVerificationFlags{
```

Comment:
I think we could set one of them to true
```
IsEthTypeChainEnabled: true,
```
I check below
```
require.True(t, flags.BlockHeaderVerificationFlags.IsEthTypeChainEnabled)
require.False(t, flags.BlockHeaderVerificationFlags.IsBtcTypeChainEnabled)
```
To ensure it is correctly set